dist/components/list/list.dust

```diff
@@ -0,0 +1,11 @@
+<ul class="c-list">
+    {#items}
+        <li class="c-list__item {modifierClass}"> {.item}
+            <ul class="c-list">
```

Comment:
I still disagree with this implementation. There is simply for prefilling the dust with nested lists. These can be added through other means.